kairon/shared/analytics/analytics_pipeline_processor.py

```
@@ -0,0 +1,163 @@
+                settings.status = False
+                settings.save()
+        except DoesNotExist:
+            raise AppException("Notification settings not found!")
+
+    @staticmethod
+    def retrieve_config(event_id, bot):
+        try:
+            settings = AnalyticsPipelineConfig.objects(id=event_id, bot=bot).get()
+            settings = settings.to_mongo().to_dict()
+            settings["_id"] = settings["_id"].__str__()
+            return settings
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)